pkg/serviceaccount/util.go

```diff
@@ -33,25 +33,26 @@ const (
 )
 
 // UserInfo returns a user.Info interface for the given namespace, service account name and UID
-func UserInfo(namespace, name, uid string) user.Info {
+func UserInfo(namespace, name, uid, tenant string) user.Info {
 	return (&ServiceAccountInfo{
 		Name:      name,
 		Namespace: namespace,
 		UID:       uid,
+		Tenant:    tenant,
 	}).UserInfo()
 }
 
 type ServiceAccountInfo struct {
-	Name, Namespace, UID string
-	PodName, PodUID      string
+	Name, Namespace, UID    string
+	PodName, PodUID, Tenant string
```

Comment:
Minor: Move Tenant to line of "Name, Namespace". They belong to the same semantic  group.

Reply:
fixed